.travis.yml

```diff
@@ -25,7 +25,7 @@ addons:
 
 matrix:
   include:
-    - python: 3.5
+    - python: 3.6
```

Comment:
It seems ``bluesky`` includes both version but from the [tag](https://github.com/NSLS-II/lightsource2-recipes/blob/master/scripts/nsls2-tag-build.sh#L52) it seems py35 is still the major platform now. I would prefer to test both, since ``travis`` is free :P